src/errors/IIErrors.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.25;
+
+interface IIErrors {
```

Comment:
Created this file to manage errors that are common for all the contracts. Is the name ok or shall we call it IICommonErrors?

Reply:
I don't know if we need this yet, I'd prefer not to have it yet. So maybe just use errors specific to your library